public/app/app.tsx

```diff
@@ -15,15 +15,15 @@ import '@szhsin/react-menu/dist/index.css';
 import ReactDOM from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { Route, Router, Switch } from 'react-router-dom';
-import { useReduxQuerySync } from './redux/useReduxQuerySync';
+import { setupReduxQuerySync } from './redux/useReduxQuerySync';
 import './sass/profile.scss';
 
 const container = document.getElementById('reactRoot') as HTMLElement;
 const root = ReactDOM.createRoot(container);
 
-function App() {
-  useReduxQuerySync();
+setupReduxQuerySync();
```

Comment:
I really don't like the idea of making this a global effect again. Let's explore a better way to make it occur earlier without

Reply:
We will explore it later.